### ISSUE TITLE
Create long-running environment for stability testing

### ISSUE THREAD

**Issue by WilliamBZA** [Particular/ServiceControl#1136]
Connects to: https://github.com/Particular/ServiceControl/issues/1130

### Scenario:

* [x] Create a docker container with the following
  * SQL Transport
  * A transport database pre-loaded with 1M audit messages
  * A way of measuring and reporting the ingestion progress - Query the SC API as well as the underlying tables and use that to determine when the raven indexes are in sync with reality?
* [ ] Add a similar pre-loaded error queue to the DB
* [ ] Change the image to have pre-ingested messages already in the RavenDB database before starting
  * Measure the impact this has
* [ ] Change the image to have the audit expiration process run whilst importing
  * Measure
* [ ] Add the ability to send RetryGroup messages at repeatable steps through the process

**Comment by WilliamBZA** [comment]
Status: I am starting with https://hub.docker.com/r/microsoft/mssql-server-windows-express/ as the base image, following up with installing and configuring ServiceControl on the image. Once that is done, I will work on loading a pre-populated database of messages

**Comment by mikeminutillo** [comment]
@WilliamBZA I have a load generator if you want it. It generates audit messages with random characteristics and bulk inserts them into a transport.

**Comment by WilliamBZA** [comment]
That'd be helpful, thanks

**Comment by SzymonPobiega** [comment]
@WilliamBZA can you updated the boxes? I guess we decided yesterday to halt this and not invest more time in the docker solution.

**Comment by WilliamBZA** [comment]
Updated. Correct, docker is on hold until the containerization process runs

**Comment by danielmarbach** [comment]
Given that containerization is on hold, for now, I'm closing this issue. @WilliamBZA feel free to add relevant information to this issue or reopen it if you think it is required.
